libvast/vast/format/json.hpp

```diff
@@ -59,8 +60,10 @@ struct default_selector {
   caf::optional<record_type> operator()(const vast::json::object& obj) const {
     std::vector<std::string> cache_entry;
     auto build_cache_entry
-      = [&cache_entry](const std::string& prefix, const vast::json&) {
-          cache_entry.emplace_back(prefix);
+      = [&cache_entry](detail::stack_vector<std::string_view, 64>& prefix,
```

Comment:
```suggestion
      = [&cache_entry](auto& prefix,
```

Because it's a messy type, I don't think it's necessary to expose it to the user.